Makefile

```diff
@@ -59,7 +59,10 @@ cluster-down:
 cluster-build:
 	./cluster/build.sh
 
-cluster-deploy:
+cluster-clean:
```

Comment:
I think this also needs to be added to `.PHONY`.

Reply:
ok, will add it.